includes/class-wc-stripe-helper.php

```diff
@@ -516,183 +476,33 @@ public static function get_legacy_payment_method( $id ) {
-					} else {
-						return str_replace( 'stripe_', '', $payment_method_id );
-					}
-				},
-				$ordered_payment_method_ids
-			);
-
-			// Cover the edge case when new Stripe payment methods are added to the plugin which do not exist in
-			// the `stripe_legacy_method_order` option.
-			if ( count( $payment_method_ids ) - 1 !== count( $payment_method_classes ) ) {
-				foreach ( $payment_method_classes as $payment_method_class ) {
-					$id = str_replace( 'stripe_', '', $payment_method_class::ID );
-					if ( ! in_array( $id, $payment_method_ids, true ) ) {
-						$payment_method_ids[] = $id;
-					}
-				}
-
-				// Update the `stripe_legacy_method_order` option with the new order including missing payment methods from the option.
-				$stripe_settings['stripe_legacy_method_order'] = $payment_method_ids;
-				self::update_main_stripe_settings( $stripe_settings );
-			}
-		} else {
-			$payment_method_ids = array_map(
-				function ( $payment_method_class ) {
-					return str_replace( 'stripe_', '', $payment_method_class::ID );
-				},
-				$payment_method_classes
-			);
-			$payment_method_ids = array_merge( [ WC_Stripe_Payment_Methods::CARD ], $payment_method_ids );
-		}
-
-		return $payment_method_ids;
+		return [];
 	}
 
 	/**
 	 * List of enabled legacy payment methods.
 	 *
 	 * @return array
+	 *
+	 * @deprecated 10.1.0 This method will be removed in future versions.
 	 */
 	public static function get_legacy_enabled_payment_methods() {
-		$payment_methods = self::get_legacy_payment_methods();
-
-		$enabled_payment_methods = [];
-
-		foreach ( $payment_methods as $payment_method ) {
-			if ( ! $payment_method->is_enabled() ) {
-				continue;
-			}
-			$enabled_payment_methods[ $payment_method->id ] = $payment_method;
-		}
-
-		return $enabled_payment_methods;
+		return [];
 	}
 
 	/**
 	 * List of enabled legacy payment method ids.
 	 *
 	 * @return array
-	 */
-	public static function get_legacy_enabled_payment_method_ids() {
-		$is_stripe_enabled = self::get_settings( null, 'enabled' );
-
-		// In legacy mode (when UPE is disabled), Stripe refers to card as payment method.
-		$enabled_payment_method_ids = 'yes' === $is_stripe_enabled ? [ WC_Stripe_Payment_Methods::CARD ] : [];
-
-		$payment_methods                   = self::get_legacy_payment_methods();
-		$mapped_enabled_payment_method_ids = [];
-
-		foreach ( $payment_methods as $payment_method ) {
-			if ( ! $payment_method->is_enabled() ) {
-				continue;
-			}
-			$payment_method_id = str_replace( 'stripe_', '', $payment_method->id );
-
-			$mapped_enabled_payment_method_ids[] = $payment_method_id;
-		}
-
-		return array_merge( $enabled_payment_method_ids, $mapped_enabled_payment_method_ids );
-	}
-
-	/**
-	 * Get settings of individual legacy payment methods.
 	 *
-	 * @return array
-	 *
-	 * @deprecated 9.6.0 The customization of individual payment methods is now deprecated.
+	 * @deprecated 10.1.0 This method will be removed in future versions.
```

Comment:
Nit: Another version fix.
```suggestion
	 * @deprecated 10.2.0 This method will be removed in future versions.
```